addons/skin.estuary/1080i/Home.xml

```diff
@@ -869,7 +864,8 @@
 				<aligny>center</aligny>
 				<onclick>ActivateWindow(1107)</onclick>
 				<font>font14</font>
-				<label>-</label>
+				<label>$LOCALIZE[31001]...</label>
```

Comment:
i would move the dots to the language file, so it can be properly translated in RTL languages
(they need the dots at the left side of the string i think...)